.github/workflows/upgrade.yml

```diff
@@ -66,6 +66,7 @@ jobs:
       - name: Process failures
         id: process-failures
         run: test -f all-failures/failure && echo 'hasFails=true' >> "$GITHUB_OUTPUT"
+        continue-on-error: true
```

Comment:
I would rather fix the script to not fail, if there are no failure artefacts, then to use the `continue-on-error`.

Can you suggest a better script?

Reply:
Makes sense, I changed the script to echo 'hasFails=true' or 'hasFails=false' depending on the result of the test command instead of ignoring failures.